frontend/src/index.test.js

```diff
@@ -1,37 +1,14 @@
-import { render, screen } from "@testing-library/react";
-import {setupLandingPage} from "./setupTests";
+import {test, expect} from '@jest/globals';
 
-describe('MyComponent tests', () => {
-  let getByTestId;
+import {makeGraph} from "./components/CsetComparisonPage";
```

Comment:
@Sigfried This is where I'm stuck right now. Seems like a JS setup problem.

If I comment this line out, my hello world test passes:
```
yarn test
yarn run v1.22.19
$ jest
 PASS  src/index.test.js
  ✓ should return the correct greeting (3 ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        0.378 s, estimated 1 s
Ran all test suites.
✨  Done in 1.02s.
```

But if I uncomment it out, I get an error:
<details><summary>Details</summary>

<p>

```
yarn test
yarn run v1.22.19
$ jest
 FAIL  src/index.test.js
  ● Test suite failed to run

    Jest encountered an unexpected token

    Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.

    Out of the box Jest supports Babel, which will be used to transform your files into valid JS based on your Babel configuration.

    By default "node_modules" folder is ignored by transformers.

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/ecmascript-modules for how to enable it.
     • If you are trying to use TypeScript, see https://jestjs.io/docs/getting-started#using-typescript
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/configuration
    For information about custom transformations, see:
    https://jestjs.io/docs/code-transformation

    Details:

    /Users/joeflack4/projects/termhub/frontend/node_modules/react-merge-refs/dist/index.mjs:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){function o(f){return r=>{f.forEach(n=>{typeof n=="function"?n(r):n!=null&&(n.current=r)})}}export{o as mergeRefs};
                                                                                                                                                                                 ^^^^^^

    SyntaxError: Unexpected token 'export'

      13 |   useDismiss,
      14 | } from "@floating-ui/react-dom-interactions";
    > 15 | import { mergeRefs } from "react-merge-refs";
         | ^
      16 |
      17 | const Tooltip = ({
      18 |   children,

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1505:14)
      at Object.require (src/components/Tooltip.jsx:15:1)
      at Object.require (src/components/dataTableUtils.jsx:4:1)
      at Object.require (src/components/CsetComparisonPage.jsx:18:1)
      at Object.require (src/index.test.js:3:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        2.412 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

</p>
</details>